drivers/gpio/gpio_lmp90xxx.c

```diff
@@ -42,15 +46,39 @@ static int gpio_lmp90xxx_config(struct device *dev, int access_op,
 		return -EINVAL;
 	}
 
-	if (flags & GPIO_INT) {
+	if ((flags & GPIO_SINGLE_ENDED) != 0) {
+		return -ENOTSUP;
+	}
+
+	if (((flags & GPIO_PULL_UP) != 0) || ((flags & GPIO_PULL_DOWN) != 0)) {
```

Comment:
This could be `(flags & (GPIO_PULL_UP | GPIO_PULL_DOWWN)) != 0U` to be a little more readable (IMO).  Or even combine all unsupported flags into a single test like that.

Reply:
Fixed, thanks.